packages/network-of-terms-graphql/README.md

```diff
@@ -0,0 +1,194 @@
+# Network of Terms GraphQL API
+
+This application delivers a [GraphQL](https://graphql.org) API for searching the Network of Terms. It is available
+at https://termennetwerk-api.netwerkdigitaalerfgoed.nl.
+
+### GraphQL endpoint
+
+If you’re a developer and want to use the Network of Terms API in your apps,
```

Comment:
Or: 'Network of Terms GraphQL API' or just 'GraphQL API' (to prevent possible confusion with the Recon API)?